packages/gravity/proto-guard/src/check-snapshot.test.ts

```diff
@@ -30,35 +31,59 @@ describe('Load client from storage snapshot', () => {
     return testStoragePath;
   };
 
-  test('check if space loads for Automerge on nodeFS snapshot', async () => {
-    const snapshot = SnapshotsRegistry.getSnapshot('automerge-nodeFS');
-    invariant(snapshot, 'Snapshot not found');
-    log.info('Testing snapshot', { snapshot });
+  test
+    .skip('check if space loads for Automerge on nodeFS snapshot', async () => {
```

Comment:
why is this test being skipped?

Reply:
its getting removed in https://github.com/dxos/dxos/pull/7031